x-pack/plugins/osquery/public/editor/index.tsx

```diff
@@ -77,7 +77,7 @@ const OsqueryEditorComponent: React.FC<OsqueryEditorProps> = ({
   );
 
   return (
-    <CodeEditor
+    <CodeEditor // TODO theme provided in context by x-pack/plugins/osquery/public/application.tsx
```

Comment:
it's done here https://github.com/elastic/kibana/blob/main/x-pack/plugins/osquery/public/application.tsx#L58

Reply:
Ok perfect! Ill remove the TODO and tag you - thanks for verifying!